Tasks/BashV3/bash.ts

```diff
@@ -72,7 +73,20 @@ async function run() {
                 targetFilePath = input_filePath;
             }
 
-            contents = `. '${targetFilePath.replace("'", "'\\''")}' ${input_arguments}`.trim();
+            if (old_source_behavior) {
```

Comment:
Instead of saying this is old can we specify as a description of what it was doing along with a note that we plan to remove it later?

Reply:
Added explanatory comment.

Reply:
We don't have a plan to remove this though